### PR TITLE
Adding documentation for token docs in Button, Card and Stack

### PR DESCRIPTION
**Pull request by khmakoto**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Adding documentation for token docs in Button, Card and Stack.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8774)

### CODE REVIEW

**Comment by khmakoto** [comment]
@msft-github-bot require review by @JasonGore or @dzearing

**Comment by khmakoto** [comment]
@msft-github-bot don't wait

**Comment by khmakoto** [comment]
@msft-github-bot wait 1 minute

File: packages/office-ui-fabric-react/src/components/Stack/Stack.types.ts [hunk 1]
```diff
@@ -135,6 +135,11 @@ export interface IStackProps
  * {@docCategory Stack}
  */
 export interface IStackTokens {
+  /**
+   * Defines the spacing between Stack children.
+   * The property is specified as a value for 'row gap', followed optionally by a value for 'column gap'.
+   * If 'column gap' is omitted, it's set to the same value as 'row gap'.
```
Comment (KevinTCoughlin) [inline packages/office-ui-fabric-react/src/components/Stack/Stack.types.ts]: Would using `@defaultValue` apply here given your doc comment?

Comment (khmakoto) [reply packages/office-ui-fabric-react/src/components/Stack/Stack.types.ts]: Hmmmm, I wonder what the `@defaultValue` would be though. Maybe just `0`? We could certainly add it if needed.

**Pull request merged**